q2_feature_table/tests/test_filter.py

```diff
@@ -203,6 +203,44 @@ def test_sample_metadata(self):
         expected = Table(np.array([]), [], [])
         self.assertEqual(actual, expected)
 
+        # exclude one
```

Comment:
Test cases added in this PR so far:
- Sample axis:
  - exclude one
  - exclude two
  - exclude all
  - incorrect invocation (this test needs to be revised, see comments below)
- Feature axis:
  - exclude one
  - exclude two
  - exclude all
  - incorrect invocation (this test needs to be revised, see comments below)

Can you add the following tests:
- Sample axis:
  - exclusion with min frequency (confirm behavior with samples above an below threshold)
  - exclusion with max frequency (confirm behavior with samples above an below threshold)
  - exclusion with min and max frequency (confirm behavior with samples above an below threshold)
  - exclusion with min features (confirm behavior with samples above an below threshold)
  - exclusion with max features (confirm behavior with samples above an below threshold)
  - exclusion with min and max features (confirm behavior with samples above an below threshold)
  - exclusion with `where` clause

I think we can get away with the tests that you have already written for the Feature axis, for the same reasons as mentioned in [the comment](https://github.com/qiime2/q2-feature-table/pull/112/files#diff-4ad12f722dbdc707bd5cb97fea327c93R388).

Let me know if you need help setting these tests up.